recipes/oauthenticator/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "oauthenticator" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f174c7b2fa20028717aa10e778f66ffa896a2fbcb2ee8a78b135d5aa8a5d56ff
+
+build:
+  number: 0
+  skip: True  # [win]
```

I think this can be a `noarch` package (it doesn't matter that it isn't supported on Windows):
```suggestion
  noarch: python
```

---

@chrisburr, thanks for the review! You were correct.